packages/p2p/src/components/modal-manager/modals/disclaimer-modal/disclaimer-modal.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Button, Checkbox, Icon, Modal, Text } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { Localize } from 'Components/i18next';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+
+const DisclaimerModal = () => {
+    const [is_checked, setIsChecked] = React.useState<boolean>(false);
+    const { hideModal, is_modal_open } = useModalManagerContext();
+    const { client, ui } = useStore();
+    const { loginid } = client;
+    const { is_mobile } = ui;
+
+    const onClickConfirm = () => {
+        const current_date = new Date().toISOString();
+        localStorage.setItem(`${loginid}_disclaimer_shown`, current_date);
+        hideModal();
+    };
+
+    return (
+        <Modal className='disclaimer-modal' is_open={is_modal_open} small has_close_icon={false}>
+            <Modal.Body className='disclaimer-modal__body'>
+                <React.Fragment>
+                    <div className='disclaimer-modal__body-title'>
+                        <Icon icon='IcWarning' size={64} className='disclaimer-modal__body-icon' />
+                        <Text
+                            as='p'
+                            color='prominent'
+                            weight='bold'
+                            line_height={is_mobile ? 'xl' : 'xxl'}
+                            size={is_mobile ? 'xs' : 's'}
+                        >
+                            <Localize i18n_default_text='For your safety:' />
+                        </Text>
+                    </div>
+                    <Text
+                        as='div'
```

Comment:
Why we are using text as div 🤔 
can't we just use div element?

Reply:
but then what about the styles?

Reply:
Don't know why you used div here but I guess you can use display: block sth which acts like a div

Reply:
i used div to get rid of the dom nesting error.. validatedomnesting error.

Reply:
Instead of passing as div can u just put the text inside div or fragment?

Reply:
yeah. that can be done.. but then i need to set the line height and font size separately

Reply:
```
<ul className="add your line height and font to this or to the li element">
   {data.map(text => <li>{text}</li>)}
</ul>
```

The best practice of this is sth like this
You don't need to use div at all
you can pass your styles to ul or li